src/components/execution/ExecutionTable.js

```diff
@@ -249,7 +249,9 @@ const mapStateToProps = ({ dataset, algorithms, executions, result }) => ({
     Boolean(dataset.getIn(['activity']).size) &&
     Boolean(algorithms.getIn(['activity']).size) &&
     Boolean(executions.get('activity').size),
-  executions: executions.getIn(['executions']),
+  executions: executions
+    .getIn(['executions'])
+    .filter(({ type }) => type === ALGORITHM_TYPES.ANONYMIZATION),
```

Comment:
This check is weird (use an algorithm type to check an execution type). We should either have separate types for executions or `type` should be contained in the `algorithm` key.

Reply:
I prefer putting `type` in `algorithm`.